src/Rule/Consignment/AllowSpecificCountriesRule.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Sdk\src\Rule\Consignment;
+
+use MyParcelNL\Sdk\src\Rule\Rule;
+
+class AllowSpecificCountriesRule extends Rule
```

Comment:
Ik denk dat `RestrictCountriesRule` of iets in die richting een duidelijkere naam is